livesql/marshal.go

```diff
@@ -0,0 +1,80 @@
+package livesql
+
+import (
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/samsarahq/thunder/sqlgen"
+	"github.com/samsarahq/thunder/thunderpb"
+)
+
+func valueToField(value interface{}) (*thunderpb.Field, error) {
+	switch column := sqlgen.MakeCanonical(value).(type) {
```

Comment:
What happens for custom types?

Reply:
Talked offline. We are going to support it by storing driver.Value. See last commit!